monailabel/scribbles/utils.py

```diff
@@ -12,13 +12,23 @@
 
 import numpy as np
 
+# torch import is needed to execute torchmaxflow
+import torch  # type:ignore
```

Comment:
This raises mypy error but is needed as standard requirement for a torch extension (which torchmaxflow is). 
Is there a way to ignore this line for mypy checks?

https://github.com/Project-MONAI/MONAILabel/runs/5859489138?check_suite_focus=true#step:8:17